packages/cli/commands/logs.js

```diff
@@ -111,6 +93,7 @@ const endpointLog = async (accountId, options) => {
 
 const appFunctionLog = async (accountId, options) => {
   const { latest, follow, compact, functionName, appPath } = options;
+  const projectName = appPath.split('/')[0];
```

Comment:
I am not sure about this from a UX perspective. Maybe worth discussing internally how this should work but one idea would be to automatically detect that we're in a project and get the `projectName` via `hsproject.json`. There could also be value in moving towards `hs project logs` and automatically detecting an app especially in the common case where there is a single app in a project.

Reply:
To add a bit more, the prefixing of the project name only comes into play when Design Manager comes into play, which is only the case for themes.

Reply:
I've added the `--projectName` argument for now as that is the easiest solution. I think we should put some thought into how to obtain this programmatically.